frontend/src/components/ActivityPanel.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import ActivityItem from './ActivityItem';
+import { ReactComponent as CloseIco } from '../assets/icons/x.svg';
```

Comment:
Can we keep some consistency with naming conventions? Lets make all icons be formatted NameIcon (this would be CloseIcon)